lib/perf_hooks.js

```diff
@@ -59,6 +59,15 @@ class Performance extends EventTarget {
       timeOrigin: this.timeOrigin,
     }, opts)}`;
   }
+
+}
+
+function toJSON() {
+  return {
+    nodeTiming: this.nodeTiming.toJSON(),
+    timeOrigin: this.timeOrigin,
+    eventLoopUtilization: this.eventLoopUtilization()
```

Comment:
I'm not sure we should include `eventLoopUtilisation` (because it's a method on `performance`, not an object property).
@jasnell wdyt?

Reply:
I just thought that it would be much more usable if a user consumes this API. Let's wait for @jasnell thoughts.

Reply:
Sorry, just now spotted the mention. I'm fine with it either way.